addon/scripts/background.js

```diff
@@ -580,15 +485,6 @@ function isFaviconUrl(url) {
     return url.indexOf("favicon") != -1;
 }
 
-// Tor seems to have an object here whereas chrome/firefox just have an id
-function getTabId(tabId) {
-    let id = tabId.id;
-    if (!id) {
-        id = tabId;
-    }
-    return id;
-}
-
 // Checks whether a transition is deemed to be bad to prevent loading subresources
 // in address bar
 function badTransition(href, type, transitionType) {
```

Comment:
Moving the functions here would require defining many more functions and variables in the global scope in .eslintrc.json. I thought for clarity it might be better to keep that list as short as possible, although I would be happy to move if you think that would be better?